src/test/crimson/seastore/test_btree_lba_manager.cc

```diff
@@ -116,10 +118,12 @@ struct btree_lba_manager_test :
   };
 
   auto create_transaction() {
-    return test_transaction_t{
+    auto t = test_transaction_t{
       lba_manager->create_transaction(),
       test_lba_mappings
     };
+    cache.alloc_new_extent<TestBlock>(*t.t, TestBlock::SIZE);
```

Comment:
@athanatos could you also note in the commit message, that the intention of this change is to mimic the real world use case where record-relative address and block-relative address are different, so that the tests exercising `LBAManager` is able to verify that `BtreeLBAManager` works without assuming that they are identical?

Reply:
Improved that commit message.